app-frontend/src/app/services/scenes/planetRepository.service.js

```diff
@@ -250,7 +251,7 @@ export default (app) => {
             });
         }
 
-        // assumes thumbnail is 256x256, which is true for planet thumbnails
+        // assumes thumbnail is 512x512, which is true for planet thumbnails
```

Comment:
should probably read "Assumes the dimensions stored in `this.thumbnailSize` are correct"